README.rst

```diff
@@ -16,7 +16,10 @@ View the published list of `Open edX Proposals (OEPs)`_ on ReadTheDocs.
 Testing locally
 ---------------
 
-To test locally in a Python virtual env::
+To test locally in a Python virtual env, you will first need to install `GraphViz <http://graphviz.org/>`_
+and either ``dot`` on the path or
+`adding options to the make call <https://groups.google.com/forum/#!topic/sphinx-users/yXgNey_0M3I>`_.
```

Comment:
The wording is a little funny here - consider splitting this into two sentences and making the tense consistent.

Reply:
I realized this is actually really only for Windows, and since generally our ecosystem doesn't support windows users I took this out. I did add in the Ubuntu command, thank you.